homeassistant/components/local_calendar/strings.json

```diff
@@ -17,7 +17,7 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     },
     "error": {
-      "invalid_ics_file": "Invalid .ics file"
+      "invalid_ics_file": "There was a problem reading the calendar information. See the error log for additional details."
```

Comment:
Should we tell them to enable debug logging, try again, and check the error log?

Reply:
Good point. The error log does at least contain the primary error message:

> Error reading the calendar information: Failed to parse calendar EVENT component: DTSTART was DATE-TIME but UNTIL was DATE: must be the same value type

Which I think is OK.  The debug log can be pretty hard to read though like the original error message shown in #141753 so I wasn't sure it was worth suggesting.